resources/views/themes/tailwind/partials/plans-minimal.blade.php

```diff
@@ -49,9 +49,15 @@
                         </div>
                     @notsubscribed
                         @subscriber
-                            <div onclick="switchPlans('{{ $plan->plan_id }}', '{{ $plan->name }}')" class="inline-flex items-center justify-center w-full px-4 py-4 text-base font-semibold text-white transition duration-150 ease-in-out @if($plan->default){{ ' bg-gradient-to-r from-wave-600 to-indigo-500 hover:from-wave-500 hover:to-indigo-400' }}@else{{ 'bg-gray-800 hover:bg-gray-700 active:bg-gray-900 focus:border-gray-900 focus:shadow-outline-gray' }}@endif border border-transparent cursor-pointer focus:outline-none disabled:opacity-25">
-                                Switch Plans
-                            </div>
+                            @if (env('CASHIER_VENDOR') == 'stripe')
```

Comment:
Using `env` directly in your project is discouraged. Wave has a [config](https://github.com/thedevdojo/wave/blob/HEAD/config/wave.php) you can add to.

A new `stripe` key for Stripe specific configurations would be better to add if you need to access Stripe specific enviroment variables.

A `billing.vendor` config option could use the `env('CASHIER_VENDOR')` so it'll be easy to check which vendor is installed by using `config('billing.vendor')`.

Reply:
Yeah, it's a good option. I choose .env because there are all the config there in just one place, but we can move it

Reply:
Also, I've built it in that way because it won't take effect with any current Wave config. So there is no conflict if you don't have any of the new Stripe .env vars

Reply:
Yeah, Laravel applications have a lot of config files. Mainly because it's the suggested approch for [retrieving enviroment variables](https://laravel.com/docs/8.x/configuration#retrieving-environment-configuration) to only use `env` in config files, and most Laravel projects follow this standard.

Most of the time you won't have to even reach into config files, you simply add the config options you want in your enviroment files. That's probably one of the reason for this two-step approach.

If you have a config that points to the enviroment variables you can update the config with new options as much as you want without conflicting with already set options.

For application specific configurations I usually create a new config file matching the project name, that keeps things nice and separated IMHO.

Reply:
> So there is no conflict if you don't have any of the new Stripe .env vars

The second parameter to the `env` function is for default values, e.g. `env('CASHIER_VENDOR', 'paddle')` would return `paddle` if `CASHIER_VENDOR` was blank or isn't set.

Reply:
So why when i downloaded wave 2.0. Stripe was not included? I saw no .env variables pointing to stripe nor any docs. 

Was this not integrated?